src/components/TicTacToe.tsx

```
@@ -0,0 +1,98 @@
+    return (
+      <button
+        className={styles.square}
+        onClick={() => handleClick(index)}
+        disabled={!!winner || !!board[index]}
+      >
+        {board[index]}
+      </button>
+    );
+  };
+
+  let status;
```

Biome (1.9.4)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)